templates/rhpam76-authoring-ha.yaml

```diff
@@ -1337,8 +1337,12 @@ objects:
           secret:
             secretName: "${KIE_SERVER_HTTPS_SECRET}"
         initContainers:
-        - command: ['sh', '-c', 'until nslookup ${APPLICATION_NAME}-mysql && nc -vz ${APPLICATION_NAME}-mysql 3306; do echo waiting for ${APPLICATION_NAME}-mysql; sleep 2; done;']
-          image: busybox
+        - command:
+            - /bin/bash
+            - '-c'
+            - >-
+              replicas=$(oc get dc ${APPLICATION_NAME}-mysql -o=jsonpath='{.status.availableReplicas}'); until "[" $replicas -gt 0 "]"; do echo waiting for ${APPLICATION_NAME}-mysql; replicas=$(oc get dc ${APPLICATION_NAME}-mysql -o=jsonpath='{.status.availableReplicas}'); sleep 2; done;
+          image: openshift4/ose-cli
```

Comment:
We just have to make sure that this image will be available in OpenShift 3.x clusters.

Reply:
I agree! I will change to version 3.x as it works for both 3.x and 4.x

Reply:
@ricardozanini Looks like @msmagnanijr has changed to 3.x, so I'm resolving this conversation.